gui/src/videodecoder.cpp

```diff
@@ -23,6 +23,7 @@
 
 VideoDecoder::VideoDecoder(ChiakiLog *log) : log(log)
 {
+  avcodec_register_all();
```

Comment:
This has been deprecated in avcodec 58.10.100, so it should be surrounded by an `#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(58, 10, 100)`.
Also, indentation must be tabs.